meta-balena-common/recipes-extended/procps/procps_%.bbappend

```diff
@@ -0,0 +1,24 @@
+
+ALTERNATIVE_${PN} = ""
+
+python procps_binpackages () {
+    def pkg_hook(f, pkg, file_regex, output_pattern, modulename):
+        pn = d.getVar('PN')
+        # d.appendVar('RRECOMMENDS_%s' % pn, ' %s' % pkg)
+
+        if d.getVar('ALTERNATIVE_' + pkg):
+            return
+        if d.getVarFlag('ALTERNATIVE_LINK_NAME', modulename):
+            d.setVar('ALTERNATIVE_' + pkg, modulename)
+
+    bindirs = sorted(list(set(d.expand("${base_sbindir} ${base_bindir} ${bindir}").split())))
+    for dir in bindirs:
+        do_split_packages(d, root=dir,
+                          file_regex='^(.*)$', output_pattern='${PN}-%s',
+                          description='${PN} %s',
+                          hook=pkg_hook, extra_depends='${PN}', allow_links=True, prepend=True)
```

Comment:
Poky's util Linux has the same binary splitting function with the addition of a special case for symlinks https://git.yoctoproject.org/cgit/cgit.cgi/poky/tree/meta/recipes-core/util-linux/util-linux.inc?id=2d0c2242136e8c823d58218844c6e082122d0bce#n49.
Is that not required in this case?

Reply:
I thought that by enabling `allow_links=True` I would achieve the same behaviour but I confirmed that wasn't the case. I've since restored the function to behave much more like util-linux.